backend/volume/src/chat/chat.gateway.ts

```diff
@@ -66,6 +66,8 @@ export class ChatGateway implements OnGatewayInit, OnGatewayConnection, OnGatewa
   async handleLoad(client: any, roomId: number) { // client verification?
     console.log('/chat/load received roomId:', roomId);
 
+    const usrs = await this.roomService.getRoomUsers(roomId);
```

Comment:
There is no reason to abbreviate "users" here. If you had to do it because users was already variable in use I would suggest to change one or both to a more meaningful name instead of abbreviating one of them!